.travis.yml

```diff
@@ -1025,9 +1131,23 @@ install:
 script:
   - |-
     echo "using $TOOLSET : : $COMPILER : <cxxflags>-std=$CXXSTD ;" > ~/user-config.jam
-  - (cd libs/config/test && ../../../b2 config_info_travis_install toolset=$TOOLSET && ./config_info_travis)
-  - (cd libs/multiprecision/test && ../../../b2 -j3 toolset=$TOOLSET $TEST_SUITE define=CI_SUPPRESS_KNOWN_ISSUES define=SLOW_COMPILER)
+  - if [[ $TRAVIS_OS_NAME == linux ]]; then
+      proc=$(($(nproc) + 1));
+    elif [[ $TRAVIS_OS_NAME == osx ]]; then
+      proc=$(($(sysctl -n hw.ncpu) + 1));
+    fi;
+  - (cd libs/config/test && ../../../b2 config_info_travis_install toolset=$TOOLSET && ./config_info_travis);
+  - (cd libs/multiprecision/test && ../../../b2 -j$proc toolset=$TOOLSET $TEST_SUITE define=CI_SUPPRESS_KNOWN_ISSUES define=SLOW_COMPILER);
+  - sudo ./b2 -j$proc install;
+  - cd $TRAVIS_BUILD_DIR && mkdir cmake-build && cd cmake-build;
+  - if [[ $(echo $CXXSTD | sed 's/[^0-9]*//g') == 03 ]]; then
+      CXXSTD=c++98;
+    fi;
+  - if [[ $CXXSTD == c++1z ]]; then
+      CXXSTD=c++17;
+    fi;
+  - cmake --target $TEST_SUITE -DCMAKE_CXX_STANDARD=$(echo $CXXSTD | sed 's/[^0-9]*//g') -DBUILD_WITH_CI_KNOWN_ISSUES_SUPPRESS=TRUE ..;
 
 notifications:
   email:
-    on_success: always
+    on_success: always
```

Comment:
Can I just get you to end this file with a newline - go ahead and let it trigger a CI build this time so I can check the (hopefully) final revision before merging.  Thanks!